CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### New features and optimizations
 - Added public-facing official Dockerfile for NeMo-Aligner
+- Added the ability to do multi-epoch (cfg.max_epochs > 1) training for reward models, DPO, and PPO
 
 ### Breaking changes
+- We have changed the shuffle logic in the data sampler for multi-epoch training, so training runs using identical parameters
```

Comment:
Just realized this may be confusing => suggested rewrite:
```suggestion
- We have changed the shuffle logic in the data sampler to support multi-epoch training, so training runs using identical parameters
```

Reply:
Sorted, thanks